recipes/r-prt/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "r-prt" %}
+{% set version = "0.1.5" %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 
+   - {{ cran_mirror }}/src/contrib/prt_{{ version }}.tar.gz
+   # - {{ cran_mirror }}/src/contrib/Archive/prt/prt_{{ version }}.tar.gz
```

Comment:
This is a fallback and shouldn't be commented out.

Reply:
There's no 0.1.5 tarball at https://cran.r-project.org/src/contrib/Archive/prt/. Not sure why.

Reply:
Tarballs are moved there whenever they stop being the latest release. It won't cause a problem to have it uncommented. Including the fallback to the archive makes the recipe flexible for maintainers so that they only need to change the version in the header and the hash to build non-latest versions.